models/taskcomment.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, Sequelize) => {
+  class TaskComment extends Model {
```

Comment:
naming of the file should be either TaskComment, User or task_comment or taskComment.

Reply:
ok